### PR TITLE
Final Critical Domain Article Edits

### PR DESCRIPTION
**Pull request by itsalyse**
Completes the updates marked `critical` in the Domain category 

Adding a domain - https://docs.google.com/document/d/1fvVOR63ckV1e_bmiV8ZKXwiUewl6v1E8BclxVxAFOJE/edit

Superlock - https://docs.google.com/document/d/1XLH9t8CokIhudBvXSUdKlbeTFfykmIRreCwjH9vzFFA/edit

.EU - https://docs.google.com/document/d/11iZIRFrumh7aG-CraTeGjC4E3-XG93N_syIdoFrJNY0/edit#heading=h.y95b8781atrj

Adding a subdomain - https://docs.google.com/document/d/1gBTCJAt0qiMbIrSonQsCVtqJ-DpyHFOPJ2VNnVbo3m8/edit

ICANN domain validation - https://docs.google.com/document/d/17gy-r7cDkO44kCAfWWkvqPaGclYMnJ-p6XNr2cAbZfs/edit#heading=h.mxy4b3kq2ep2

Belongs to https://github.com/dnsimple/dnsimple-business/issues/1900

### CODE REVIEW

File: content/articles/domains-eu.md [hunk 1]
```diff
@@ -14,23 +14,26 @@ categories:
...
-  - A natural person who is not an EU citizen and who is a resident of a Member State.
+  - An EU citizen, independent of their place of residence.
+  - A natural person who is not an EU citizen, and who is a resident of a Member State.
   - An undertaking that is established in the EU.
   - An organization that is established in the EU, without prejudice to the application of the national law.
   
 If your residential address isn't based in the EU, you must select and self-declare your country of citizenship from the dropdown.
 
-![Register .EU domain](/files/register-eu-domain.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/domains-eu.md:35]: https://drive.google.com/file/d/1FO_6fzw84Rky_vr27aHRpBFe5WYXeYJd/view?usp=sharing

File: content/articles/adding-subdomain.md [hunk 2]
```diff
@@ -7,26 +7,29 @@ categories:
...
-1.  Click on the <label>DNS</label> tab and scroll to the record card (or quickly jump to the DNS records for that domain by clicking the button in the top right).
+1.  On the header, click the <label>Domain Names</label> tab.
+1.  Locate the relevant domain, and click on the name to access the domain page.
+1.  Click the <label>DNS</label> tab, and scroll to the **DNS Records** card, or click the **DNS** badge at the top right to jump to the DNS records for that domain.
 
-    ![Access DNS records](/files/example-domain-manage.png)
+<!--- needs screenshot -->
 
-1.  From there, click the <label>Add record</label> drop-down button, and select the type of record you want to create. [`A`, `ALIAS`, and `CNAME` records](/articles/differences-between-a-cname-alias-url) will all create a new subdomain.
+1.  Click <label>Add record</label>, and select the type of record you want to create. [`A`, `ALIAS`, and `CNAME` records](/articles/differences-between-a-cname-alias-url) will all create a new subdomain.
```
Comment (itsalyse) [inline content/articles/adding-subdomain.md:22]: Check numbering

File: content/articles/adding-subdomain.md [hunk 3]
```diff
@@ -7,26 +7,29 @@ categories:
...
 
-    ![Custom Records](/files/example-domain-dns.jpg)
+<!--- needs screenshot -->
 
-    That will take you the advanced editor where you can add the records for the subdomain.
+    Choose a record to open advanced editor where you can add the records for the subdomain.
 
-1.  To add a subdomain, just fill in the subdomain in the Name field. For example, `www` would be the subdomain `www.example.com`.
+1.  To add a subdomain, fill in the subdomain in the **Name** field. For example, `www` would be the subdomain `www.example.com`.
```
Comment (itsalyse) [inline content/articles/adding-subdomain.md:28]: Check numbering

File: content/articles/adding-subdomain.md [hunk 4]
```diff
@@ -7,26 +7,29 @@ categories:
...
 
-    ![Add Record](/files/add-a-record.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/adding-subdomain.md:30]: https://drive.google.com/file/d/195vVDp5io2k4FsJ0s6plxdvqwNDsWcCP/view?usp=sharing

File: content/articles/adding-subdomain.md [hunk 5]
```diff
@@ -7,26 +7,29 @@ categories:
...
+1.  Click the <label>DNS</label> tab, and scroll to the **DNS Records** card, or click the **DNS** badge at the top right to jump to the DNS records for that domain.
 
-    ![Access DNS records](/files/example-domain-manage.png)
+<!--- needs screenshot -->
 
-1.  From there, click the <label>Add record</label> drop-down button, and select the type of record you want to create. [`A`, `ALIAS`, and `CNAME` records](/articles/differences-between-a-cname-alias-url) will all create a new subdomain.
+1.  Click <label>Add record</label>, and select the type of record you want to create. [`A`, `ALIAS`, and `CNAME` records](/articles/differences-between-a-cname-alias-url) will all create a new subdomain.
 
-    ![Custom Records](/files/example-domain-dns.jpg)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/adding-subdomain.md:24]: https://drive.google.com/file/d/195vVDp5io2k4FsJ0s6plxdvqwNDsWcCP/view?usp=sharing

File: content/articles/adding-subdomain.md [hunk 6]
```diff
@@ -7,26 +7,29 @@ categories:
...
 1.  If you have more than one account, select the relevant one.
-1.  On the header, click the <label>Domains</label> tab.
-1.  Locate the relevant domain and click on the name to access the domain page.
-1.  Click on the <label>DNS</label> tab and scroll to the record card (or quickly jump to the DNS records for that domain by clicking the button in the top right).
+1.  On the header, click the <label>Domain Names</label> tab.
+1.  Locate the relevant domain, and click on the name to access the domain page.
+1.  Click the <label>DNS</label> tab, and scroll to the **DNS Records** card, or click the **DNS** badge at the top right to jump to the DNS records for that domain.
 
-    ![Access DNS records](/files/example-domain-manage.png)
+<!--- needs screenshot -->
```
Comment (itsalyse) [inline content/articles/adding-subdomain.md:20]: https://drive.google.com/file/d/1CYyPKlhv_LOehki4R49fLQSFydpE-24b/view?usp=sharing

File: content/articles/icann-domain-validation.md [hunk 7]
```diff
@@ -29,11 +30,13 @@ The link must be to `raa.name-services.com`, `www.enom.com`, `registrant-email-v
...
+If you are having trouble finding the email verification in your inbox, please remember to check your spam. 
+
+If it isn't there, resend the email by navigating to your domain page, and locating the ICANN pending verification warning at the top of the page. Click **Send ICANN RAA verification email**.
 
 ![ICANN resend email verification](/files/raa-resend-email-verification.png)
 
-Once you have acted on the verification email, the warning above should not appear. If it does, it may be because our system hasn't refreshed the latest status yet. You can manually force a refresh by clicking on the "Refresh" button.
+Once the email is verified, the warning should disappear. If it doesn't, it may be because our system hasn’t refreshed the latest status yet. Click **Refresh** to manually force a refresh. 
 
 ## What happens when a domain is suspended?
```
Comment (itsalyse) [inline content/articles/icann-domain-validation.md:42]: can add the centralnic ones as soon as we get them, or if  that's going to take a bit, we can push the major changes and then make a note to revisit this when we have those

Comment (stephanoodle) [reply content/articles/icann-domain-validation.md:42]: I think we should push the changes now and add CentralNic later. We're still figuring out the CentralNic process for this. I discovered today that a lot of it is very different.

**Review by stephanoodle — APPROVED** [review]

File: content/articles/icann-domain-validation.md (hunk 7 above)
Comment (Pemacaa) [reply content/articles/icann-domain-validation.md:42]: @itsalyse got the CentralNic nameservers: ns1.emailverification.info
ns2.emailverification.info

**Review by Pemacaa — APPROVED** [review]

**Comment by dperrymorrow** [comment]
@itsalyse  I have added the screenshots, please verify on the preview and ill deploy

**Pull request merged**